internal/flink/command_environment_create.go

```diff
@@ -26,6 +27,7 @@ func (c *command) newEnvironmentCreateCommand() *cobra.Command {
 	cmd.Flags().String("defaults", "", "JSON string defining the environment's Flink application defaults, or path to a file to read defaults from (with .yml, .yaml or .json extension).")
 
 	addCmfFlagSet(cmd)
+	pcmd.AddOutputFlag(cmd)
```

Comment:
I think we might want to create another version of this flag that just has json and yaml outputs.

That way, the flag description won't show human output as option, and the command won't error by default.

Reply:
Environment allows all the formats i.e JSON, YAML and human.
For application, I have created another registration function `AddOutputFlagWithHumanRestricted` where the allowed values are only JSON and YAML. This ensures that for application command we do not print help like 

```
Specify the output format as "human", "json", or "yaml". (default "json")
``` 
when we do not support human

Reply:
Logic behind the decision to continue `human` output for env commands: The version included in CP release has only human output create, update and describe environment
While I personally don't like the human output, I would like to not remove things in the future version of the CLI (unless it is v5), so with the change I am making I will simply add support for JSON and YAML and not take away what is already there

Reply:
@sgagniere I have updated the description, please take a look